public/js/followcube.js

```diff
@@ -0,0 +1,11 @@
+const follow = async (cubeID) => {
+  await fetch(`/cube/api/follow/${cubeID}`);
+  await console.log(`followed ${cubeID}`);
+  await location.reload();
+}
+
+const unfollow = async (cubeID) => {
+  await fetch(`/cube/api/unfollow/${cubeID}`);
+  await console.log(`unfollowed ${cubeID}`);
+  await location.reload();
+}
```

Comment:
shouldn't need to `await location.reload()` because you're not doing anything after it, also remove console.log